services/QuillLMS/app/workers/save_user_pack_sequence_items_worker.rb

```diff
@@ -4,7 +4,9 @@ class SaveUserPackSequenceItemsWorker
   include Sidekiq::Worker
 
   sidekiq_options queue: SidekiqQueue::DEFAULT,
-    lock: :until_executed
+    lock: :until_executed,
```

Comment:
Note, your config options here are valid, but some combinations of config options can be invalid, so the gem provides [built-in tests](https://github.com/mhenrixon/sidekiq-unique-jobs#validating-worker-configuration) to check for that. Probably worth seeing if these tests work as a backstop against future changes.
![Screenshot 2023-01-31 at 4 35 09 PM](https://user-images.githubusercontent.com/1304933/215888630-a0e19479-3149-43f4-b979-569debafd600.png)

Reply:
Good idea.  I didn't even notice this part of the docs.   I think I was stuck on the section right below on uniqueness:  "This has been probably the most confusing part of this gem. People get really confused with how unreliable the unique jobs have been."